12.nadeo-importer/03.how-to-make-a-specularmap-diffusemap-for-objects/docs.md

```diff
@@ -15,8 +16,27 @@ I will explain very quickly what are the specifications and how to save them in
 
 ## What you need
 
-* a graphic manipulation software (ps, gimp etc.)
+* a graphic manipulation software (Adobe Photoshop (PS), Gimp etc.)
+Free & Open Source: https://www.gimp.org/
+Free Trial: https://www.adobe.com/products/photoshop.html
+
 * a dds converter which allows you to set the specific output format (like the nvidia dds tools)
+For the Gimp .dds plugin search google.
+PS Plugin: https://developer.nvidia.com/nvidia-texture-tools-adobe-photoshop
+
+
+## Compare files with originals
+
+Sometimes it is useful to see what nadeo used, so you need the original files.
+They are stored as .zip file in C:\ProgramData\ManiaPlanet\PacksCache
```

Comment:
It's better to put this in backticks, like this:
```
`C:\ProgramData\ManiaPlanet\PacksCache`
```
This would look like this: `C:\ProgramData\ManiaPlanet\PacksCache`